packages/blade/src/components/Form/Selector/SelectorGroupField.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import type { CSSObject } from 'styled-components';
+import { getPlatformType, makeAccessible } from '~utils';
+import Box from '~components/Box';
+
+type SelectorGroupFieldProps = {
+  children: React.ReactNode;
+  labelledBy: string;
```

Comment:
Can we stick to american english since we're doing that everywhere? 😅 
```suggestion
  labeledBy: string;
```

Reply:
Or are we doing this cuz of a11y props?

Reply:
> Can we stick to american english since we're doing that everywhere? 😅

our `makeAccessible` follows the WAI-ARIA props which takes british `labelledBy`, let's keep it as is for consistency sake. 

> Or are we doing this cuz of a11y props?

Yes so that the group has a label reference of it's child text. It works similarly to `fieldset` & `legend`

```tsx
<fieldset>
  <legend>hello</legend>
</fieldset>

<SelectorGroupField labelledBy="id-1">
  <Text id="id-1">hello</Text>
</SelectorGroupField>
```